src/containers/InvoiceDetails/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Table } from 'antd';
+import _ from 'lodash';
+import { Outlet, Route, Routes, useParams } from 'react-router-dom';
+
+import { RenderRemoteData } from 'aidbox-react/lib/components/RenderRemoteData';
+
+import { BasePageContent } from 'src/components/BaseLayout';
+
+import { InvoiceDetailsHeader } from './components/InvoiceDetailsHeader';
+import { useInvoiceDetails, useInvoiceLineItems } from './hooks';
+import { InvoiceDetailsLineItemsProps } from './types';
+import { formatMoney } from '../InvoiceList/utils';
+
+export function InvoiceDetails() {
+    const params = useParams();
+    const { invoiceDetailsResponse } = useInvoiceDetails(params.id!);
+
+    return (
+        <RenderRemoteData remoteData={invoiceDetailsResponse}>
+            {({ invoice, patient, practitioner }) => (
+                <>
+                    <InvoiceDetailsHeader invoice={invoice} patient={patient} practitioner={practitioner} />
+                    <BasePageContent>
+                        <Routes>
+                            <Route
+                                path="/"
+                                element={
+                                    <>
+                                        <Outlet />
+                                    </>
+                                }
+                            >
+                                <Route path="/" element={<LineItemsTable invoice={invoice} />} />
+                            </Route>
+                        </Routes>
+                    </BasePageContent>
+                </>
+            )}
+        </RenderRemoteData>
+    );
+}
+
+function LineItemsTable(props: InvoiceDetailsLineItemsProps) {
+    const [response] = useInvoiceLineItems(props);
+    const lineItems = props.invoice.lineItem?.map((lineItem) => {
+        return {
+            id: lineItem.chargeItemReference?.reference?.split('/')[1],
```

Comment:
There are a lot os places where I want to get an id from the FHIR reference... Maybe I should add a helper function?